pydantic_ai_slim/pydantic_ai/models/google.py

```diff
@@ -513,7 +529,18 @@ def _process_response_from_parts(
 ) -> ModelResponse:
     items: list[ModelResponsePart] = []
     for part in parts:
-        if part.text is not None:
+        if part.executable_code is not None:
+            items.append(ServerToolCallPart(args=part.executable_code.model_dump(), tool_name='code_execution'))
+        elif part.code_execution_result is not None:
+            # TODO(Marcelo): Is the idea to generate the tool_call_id on the `executable_code`, and then pass it here?
```

Comment:
I feel like we can/should answer this question before merging?

Reply:
Let's remove this comment as the value is never sent back to Google anyway